namer/consul/src/main/scala/io/buoyant/namer/consul/ConsulInitializer.scala

```diff
@@ -50,11 +52,16 @@ case class ConsulConfig(
    */
   @JsonIgnore
   def newNamer(params: Stack.Params): CatalogNamer = {
+    val filters = if (token.isDefined)
```

Comment:
it's idiomatic to avoid `Option.get`:

```
val filters = token match {
  case Some(t) => new SetHostFilter(...) andThen new SetAuthTokenFilter(t)
  case None => ...
}
```
or
```
val authFilter = token match {
  case Some(t) => new SetAuthTokenFilter(t)
  case None => Filter.identity
}
val filters = new SetHostFilter(...) andThen authFilter
```